pkg/placement/membership.go

```diff
@@ -33,6 +36,7 @@ const (
 	// raftApplyCommandMaxConcurrency is the max concurrency to apply command log to raft.
 	raftApplyCommandMaxConcurrency = 10
 	barrierWriteTimeout            = 2 * time.Minute
+	GRPCContextKeyExpectsVNodes    = "dapr-expects-vnodes"
```

Comment:
Not a fan of using `expects` as the client verb. Can we have something like `dapr-accept-vnodes`?.